### DISCUSSION TITLE
MacOS - ACES IDT Folder

### DISCUSSION THREAD

**Discussion by scubadircks** [xtremestuff/protune-transforms#2] [General]
Thank you for these. Using GoPro for a lot of underwater footage and these will save a lot of time.  

For DaVinci Studio 17.4.6 on MacOS, I do not have an ACES Transforms / IDT folder as per below:

~/Library/Application Support/Blackmagic Design/DaVinci Resolve/ACES Transforms/IDT/.  

Not sure if I need to create the folder or if DaVinci moved it around.  LUTs work great though.

(An intermediate user and first time delving into bowels of ACES... so forgive my ignorance if I am missing something simple.  Thanks for your work and support!)

<img width="688" alt="Screen Shot 2022-06-27 at 11 44 02 PM" src="https://user-images.githubusercontent.com/108313647/176080767-1ca6e226-3dd9-463b-8fc4-ed162b8f3efd.png">

**Comment by RikshaDriver** [comment]
Does the ACES Transform folder exist under: ~/Library/Application Support/Blackmagic Design/DaVinci Resolve/ACES Transforms

If so, then you can simply create the IDT folder and  copy the IDTs.

**Comment by scubadircks** [comment]
There is no ACES Transform folder at all.  Will try and manually add both
and see what happens.

**Reply by dutchct** [reply]
Did this ever work for you? I've just tried as well and can't see any IDT's or ODT's in resolve.

**Reply by RikshaDriver** [reply]
/Library and ~/Library are not the same.

The tilde (~) symbol represents the home directory.

~/Library expands to /Users/YourUsername/Library on MacOS

**Comment by RikshaDriver** [comment]
I have added some crude script-based installers for macOS/Linux and Windows in the latest release. This should help with installing to the correct folder.

**Comment by dutchct** [comment]
Thank you, this worked.
